python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -243,6 +245,37 @@ def getExecutionOrder(cls) -> float:
         # Docstring inherited.
         return cls.FLUX_ORDER
 
+    def _computeACF(self, kernel: lsst.afw.math.Kernel) -> lsst.afw.image.Image:
```

Comment:
Can you put a docstring here?  I guess this is "compute auto-correlation function"?